### PR TITLE
TRUNK-5367: Create URL global property on OpenMRS Core

### PR DESCRIPTION
**Pull request by Elbertbiggs360**
### JIRA TICKET NAME
[TRUNK:5367: Create URL global property on OpenMRS Core](https://issues.openmrs.org/browse/TRUNK-5367)
## Description of what I changed
This property should hold the default login URL and be publicly accessible

## Issue I worked on
[TRUNK:5367](https://issues.openmrs.org/browse/TRUNK-5367)
## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by Charpell** [comment]
Looks good to me

File: api/src/main/java/org/openmrs/util/OpenmrsConstants.java [hunk 1]
```diff
@@ -1078,6 +1078,9 @@ public static String getOpenmrsProperty(String property) {
 		
 		props.add(new GlobalProperty("allergy.concept.unknown", "1067AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
 		        "UUID for the allergy unknown concept"));
+
+		props.add(new GlobalProperty(LOGIN_OWA_URL, "/openmrs/login.htm",
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: The context name will not always be openmrs

File: api/src/main/java/org/openmrs/util/OpenmrsConstants.java [hunk 2]
```diff
@@ -1234,6 +1237,8 @@ public static String getOpenmrsProperty(String property) {
 	public static final String LOG_LEVEL_ERROR = "error";
 	
 	public static final String LOG_LEVEL_FATAL = "fatal";
+
+	public static final String LOGIN_OWA_URL = "/openmrs/login.htm";
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: The context name will not always be openmrs

File: api/src/main/java/org/openmrs/util/OpenmrsConstants.java (hunk 1 above)
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: The name LOGIN_OWA_URL means that this cannot be handled by anything other than OWAs

File: api/src/main/java/org/openmrs/util/OpenmrsConstants.java [hunk 3]
```diff
@@ -1078,6 +1078,9 @@ public static String getOpenmrsProperty(String property) {
 		
 		props.add(new GlobalProperty("allergy.concept.unknown", "1067AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
 		        "UUID for the allergy unknown concept"));
+
+		props.add(new GlobalProperty(LOGIN_URL, "../../login.htm",
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: Why not just login.htm?

**Comment by dkayiwa** [comment]
Closed because the author is no longer working on the ticket.

**Pull request closed without merging**